kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriter.java

```diff
@@ -189,6 +186,7 @@ private boolean isPartialFailure(SortedMap<SinkRecord, InsertAllRequest.RowToIns
     for (Map.Entry<SinkRecord, InsertAllRequest.RowToInsert> row: rows.entrySet()) {
       if (failRowsSet.contains((long)index)) {
         failRows.put(row.getKey(), row.getValue());
+        logger.trace("Failed Record: {}", row);
```

Comment:
After digging a little, this is the import point to us. We really need to know more about failed records. I don't need to relate them with record pooled by Kafka Connect.

This simplify changes I guess.